config/server.js

```diff
@@ -10,7 +10,7 @@ const config = {
     pgUri: process.env.DATABASE_URL || 'postgres://localhost:5432/concaria'
   },
   habanero: {
-    host: process.env.HABANERO_HOST
+    embedded: process.env.HABANERO_EMBEDDED || 'true'
```

Comment:
I think it's better to convert the string to a boolean here, so we can use it as a boolean everywhere.